resources/views/choices/index.blade.php

```diff
@@ -1,46 +1,45 @@
+                                                    class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                                            </button>
+                                        </a>
+                                    </td>
+                                    <td>
+                                        <form action="{{ route('choices.destroy', $choice->id)}}" method="post">
+                                            @csrf
+                                            @method('DELETE')
+                                            <button class="btn btn-danger" type="submit"
+                                                    onclick="return confirm('Are you sure you want to delete?')">Delete
+                                            </button>
+                                        </form>
+                                    </td>
                                 </tr>
-                                </thead>
-                                <tbody>
-                                @foreach($choices as $choice)
-                                    <tr>
-                                        <td>{{ $choice->name }}</td>
-                                        <td>
-                                            <a href="{{ route('choices.edit', $choice->id) }}"
-                                               title="edit scenario">
-                                                <button class="btn btn-primary btn-sm"><i
-                                                        class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
-                                                </button>
-                                            </a>
-                                        </td>
-                                        <td>
-                                            <form action="{{ route('choices.destroy', $choice->id)}}" method="post">
-                                                @csrf
-                                                @method('DELETE')
-                                                <button class="btn btn-danger" type="submit"
-                                                        onclick="return confirm('Are you sure you want to delete?')">Delete
-                                                </button>
-                                            </form>
-                                        </td>
-                                    </tr>
-                                @endforeach
-                                </tbody>
-                            </table>
-                            {{--                    {{ $scenarios->links('pagination::bootstrap-4')  }}--}}
-                        </div>
+                            @endforeach
+                            </tbody>
+                        </table>
+                        {{ $choices->links('pagination::bootstrap-4')  }}
```

Comment:
misschien mooier om in de AppServiceProvider Paginator::useBootstrapFour(); te zetten?
[https://laravel.com/docs/9.x/pagination#using-bootstrap](https://laravel.com/docs/9.x/pagination#using-bootstrap)